docs/fairlearn.exceptions.rst

```diff
@@ -1,7 +0,0 @@
-fairlearn\.exceptions module
-----------------------------
-
-.. automodule:: fairlearn.exceptions
```

Comment:
there's some reason why this exists. I think it doesn't show up in the doc otherwise? @riedgar-ms

Reply:
Yes. That's the directive that tells Sphinx to open up the exceptions module and document it.

Reply:
I thought it was removed after we moved to sklearn's NotFittedError, I'll double check.

Reply:
You may very well be right on that one!